src/openforms/forms/api/serializers/form.py

```diff
@@ -317,6 +320,10 @@ def get_required_fields_with_asterisk(self, obj) -> bool:
         config = GlobalConfiguration.get_solo()
         return config.form_display_required_with_asterisk
 
+    def get_resume_link_lifetime(self, obj) -> int:
+        config = GlobalConfiguration.get_solo()
+        return config.incomplete_submissions_removal_limit
```

Comment:
this is also specified on the form itself: https://github.com/open-formulieren/open-forms/blob/master/src/openforms/forms/models/form.py#L216 and possibly `all_submissions_removal_limit` plays a role in this too.